go/vt/vtgate/evalengine/expressions.go

```diff
@@ -251,21 +251,21 @@ func (t TupleExpr) Evaluate(env *ExpressionEnv) (EvalResult, error) {
 }
 
 // Evaluate implements the Expr interface
-func (b *BindVariable) Evaluate(env *ExpressionEnv) (EvalResult, error) {
-	val, ok := env.BindVars[b.Key]
+func (bv *BindVariable) Evaluate(env *ExpressionEnv) (EvalResult, error) {
```

Comment:
Changes to this file seem unrelated. Are they really required?

Reply:
Oh this was due to the new linter throwing errors in that file. I'll see what I can do here.

Reply:
ok this at least was already merged by another PR